shared/tests/utility_module/actor_test.go

```diff
@@ -71,15 +71,16 @@ func TestUtilityContext_HandleMessageStakeApp(t *testing.T) {
 func TestUtilityContext_HandleMessageEditStakeApp(t *testing.T) {
 	ctx := NewTestingUtilityContext(t, 0)
 	actor := GetAllTestingApps(t, ctx)[0]
-	msg := &typesUtil.MessageEditStakeApp{
-		Address:     actor.Address,
-		Chains:      defaultTestingChains,
-		AmountToAdd: zeroAmountString,
-		Signer:      actor.Address,
+	msg := &typesUtil.MessageEditStake{
+		Address:   actor.Address,
+		Chains:    defaultTestingChains,
+		Amount:    defaultAmountString,
+		Signer:    actor.Address,
+		ActorType: typesUtil.ActorType_App,
 	}
 	msgChainsEdited := msg
 	msgChainsEdited.Chains = defaultTestingChainsEdited
-	if err := ctx.HandleMessageEditStakeApp(msgChainsEdited); err != nil {
+	if err := ctx.HandleEditStakeMessage(msgChainsEdited); err != nil {
 		t.Fatal(err)
 	}
 	actor = GetAllTestingApps(t, ctx)[0]
```

Comment:
DItto fro all other Get calls in the file.

Should this be `GetAllTestingApps`  or something more general?

Reply:
I'm using apps here, but could be any actor